export async function getError(api, page) {
    return await api.fetchError(page)
}